labs/SimonSpeck/SimonSpeck.md

```diff
@@ -375,7 +375,7 @@ able to load the `SpeckTestVectors` module also located in the `Speck`
 folder as follows and verify that you have correctly implemented the
 functions:
 
-```Xcryptol-session ci-none
+```Xcryptol-session-ci-none
```

Comment:
Same here. The tests pass pretty quickly in the comment. Does the CI need to ignore this?

Reply:
I'll remove them and see how CI does.

Reply:
Some of the proofs stall for different Cryptol or solver versions.

Reply:
That escalated quickly.

Reply:
We should decide whether we want exercises in `Basics.md`.  If so, we should do the same thing we do for most other exercises: fill them in in `BasicsAnswers.md` and verify our answers.